Tests/L0/SonarQubePreBuild/CreateCommandLineArgs.ps1

```diff
@@ -0,0 +1,65 @@
+[CmdletBinding()]
```

Comment:
The bug fix is trivial and it is a good oportunity to add unit tests. The build team have created an infrastructure for this. They have an overview here: https://github.com/Microsoft/vso-agent-tasks/blob/master/docs/runningtests.md

These are L0 tests, i.e. unit tests that require neither a server nor a build agent to run. Existing tests tend to cover either units (i.e. functions) or the whole thing (i.e. integration tests that run E2E, using a lot of mocks). These tests are unit tests.